### DISCUSSION TITLE
Error in logs when deploying Kong on GCP Cloud Run but Kong still running

### DISCUSSION THREAD

**Discussion by pidanou** [Kong/kong#11649] [Help]
Hi, 
I am trying to deploy an API and running almost exclusively on serverless services on GCP.
I deployed Kong Gateway on Cloud Run to have a single entrypoint on all my serverless functions.

At first everything seems fine and Kong is working well, the only thing is that I can see in the logs:
`worker.lua:138: communicate(): failed to connect: failed to receive response header: timeout, context: ngx.timer`

I searched around the repo and couldn't find the source of the error that doesn't seem to be blocking. It is flooding my logs though.

How can I get rid of this error?

More info:
- I'm running the latest Docker image
- I'm using DB-Less configuration
- I'm injecting the kong.yml config at startup using this command: `eval "echo \"$(cat /tmp/kong_tmp.yml)\"" > /tmp/kong.yml && /docker-entrypoint.sh kong docker-start`
- I only expose port 8000 as Cloud Run only allows single port exposition
- All the outbound trafic from Kong is sent to a serverless VPC connector because all my serverless functions allow only internal traffic from VPC
- the VPC where the outbound trafic is sent to has an external IP

**Comment by StarlightIbuki** [comment]
The logs look like from a hybrid deployment rather than a DBless.
Could you share the version number of the Kong you were using?

**Comment by joe1981al** [comment]
@pidanou did you find a resolution for this?
